Code/GraphMol/QueryOps.h

```diff
@@ -121,21 +121,21 @@ static inline int makeAtomType(int atomic_num, bool aromatic) {
   return atomic_num + 1000 * static_cast<int>(aromatic);
 }
 static inline void parseAtomType(int val, int &atomic_num, bool &aromatic) {
-  if(val>1000) {
+  if (val > 1000) {
```

Comment:
An aside: I really should do another `clang-tidy` pass across the codebase. Maybe a ToDo for just before/after the next release.

Reply:
How about we run clang-tidy before every release?

Reply:
That's a great idea.
I can run the python formatting thing at the same time.